corehq/apps/es/tests/test_esquery.py

```diff
@@ -141,28 +141,9 @@ def test_filtered_forms(self):
                 .xmlns('banana')
         self.checkQuery(query, json_output)
 
-    @patch('corehq.apps.locations.models.SQLLocation.objects.get_locations_and_children_ids')
-    def test_users_at_locations_and_descendants(self, locations_patch):
```

Comment:
Because the sql call was being patched this was basically testing the same thing that `test_users_at_locations` tests